velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -100,6 +100,21 @@ class SubstraitVeloxExprConverter {
   /// The map storing the relations between the function id and the function
   /// name.
   std::unordered_map<uint64_t, std::string> functionMap_;
+
+  // The map storing the Substrait extract function input field and velox function name.
+  std::unordered_map<std::string, std::string> extractDatetimeFunctionMap_ = {
+    {"MILLISECOND", "millisecond"},
+    {"SECOND", "second"},
+    {"MINUTE", "minute"},
+    {"HOUR", "hour"},
+    {"DAY", "day"},
+    {"DAY_OF_WEEK", "day_of_week"},
+    {"DAY_OF_YEAR", "day_of_year"},
+    {"MONTH", "month"},
+    {"QUARTER", "quarter"},
+    {"YEAR", "year"},
+    {"YEAR_OF_WEEK", "year_of_week"}
```

Comment:
Hi, are these names all for extract function? Maybe they should be added to Substrait at below place.

https://github.com/substrait-io/substrait/blob/main/extensions/functions_datetime.yaml#L10

Reply:
You're right, I will open a PR to add these names.